src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -337,6 +337,24 @@ class WelcomePage extends Disposable {
 		this.createDropDown();
 		this.createWidePreviewToolTip();
 		this.createPreviewModal();
+		this.handleAccessibility(this.commandService);
+	}
+
+	private async handleAccessibility(commandService: ICommandService): Promise<void> {
+		const tileServer = document.querySelector('#tile-server-link') as HTMLElement;
+
+		addStandardDisposableListener(tileServer, 'keydown', event => {
+			if (event.equals(KeyCode.Enter)) {
```

Comment:
should also treat Space key as Enter

Reply:
@alanrenmsft fixed

Reply:
i only see Enter key here, am i missing something?